driver-core/src/test/resources/retryable-writes/insertOne-serverErrors.json

```diff
@@ -541,6 +571,58 @@
             "insert"
           ],
           "errorCode": 89,
+          "errorLabels": [
+            "RetryableWriteError"
+          ],
+          "closeConnection": false
+        }
+      },
+      "operation": {
+        "name": "insertOne",
+        "arguments": {
+          "document": {
+            "_id": 3,
+            "x": 33
+          }
+        }
+      },
+      "outcome": {
+        "result": {
+          "insertedId": 3
+        },
+        "collection": {
+          "data": [
+            {
+              "_id": 1,
+              "x": 11
+            },
+            {
+              "_id": 2,
+              "x": 22
+            },
+            {
+              "_id": 3,
+              "x": 33
+            }
+          ]
+        }
+      }
+    },
+    {
+      "description": "InsertOne succeeds after ExceededTimeLimit",
```

Comment:
Is my understanding correct that this test is the only one really relevant for the code change (and only the error code part, not the error label part)? Are the others just pulled in because you grabbed latest from the specs repo?

Reply:
yeap - these are the latest json tests.